alerts/openshift-virtualization-operator/PodSchedulingMightBeImbalancedAcrossNodes.md

```diff
@@ -0,0 +1,22 @@
+# PodSchedulingMightBeImbalancedAcrossNodes
+
+## Meaning
+
+This alert is applicable to clusters that have a large number of container
+images per node. It is triggered when a node exceeds the maximum number of
+configured images (default: 50). Once this limit is reached, the scheduler will
+be unable to schedule any new pods that require new images on that node.
+
+## Impact
+
+Pod scheduling might be imbalanced across nodes.
+
+## Mitigation
+
+As a workaround, you can edit the `nodeStatusMaxImages` parameter of the
+`KubeletConfig` object, either by increasing its value or by setting its value
+to -1 to disable the image limit.
```

Comment:
Much better wording! Just put backticks around the value: `-1`

Reply:
fixed